### PR TITLE
Fix authentication command in README.

### PR DESCRIPTION
**Pull request by LyonesGamer**
Follow-up on #247, fix a minor typo in one line.

### CODE REVIEW

**Comment by ElijahCaine** [comment]
+1

**Comment by Kennric** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ If you have a user instance already, then simply first authenticate (italics
 indicate user input):
 
 ```
-$ curl -XPOST -d '{"auth":{"type":"password","_username_":"user","password":"_pass_"}}'
+$ curl -XPOST -d '{"auth":{"type":"password","_username_":"user","password":"_pass_"}}' -H "Content-Type: application/json" http://localhost:8000/v1/login
 ```
```
Comment (iCHAIT) [inline README.md]: @LyonesGamer 

I feel that this part of code should be like this - 

> If you have a user instance already, then simply first authenticate (angular brackets indicates user input):

```
$ curl -XPOST -d '{"auth":{"type":"password","username":"<user>","password":"<pass>"}}'
```

While setting up,  I was actually having this particular issue, I think this part is a bit confusing as `italics` is not actually getting rendered in the code segment.

Thoughts?

**Comment by LyonesGamer** [comment]
Yeah, I didn't think about the fact that formatting would be escaped inside of a code block. I've changed it to use angle brackets instead.

**Pull request merged**